scripts/zkai_native_seq32_attention_mlp_single_proof_gate.py

```
@@ -0,0 +1,684 @@
+    proof_bytes = metrics["proof_bytes"]
+    if not input_value.get("validation_commands"):
+        raise NativeSeq32AttentionMlpSingleProofGateError("input validation commands missing")
+    if not metrics["envelope"].get("proof_backend") == "stwo":
+        raise NativeSeq32AttentionMlpSingleProofGateError("envelope backend drift")
+    if len(proof_bytes) != row.get("proof_json_size_bytes"):
+        raise NativeSeq32AttentionMlpSingleProofGateError("proof payload byte length drift")
+    if sha256(proof_bytes) != row.get("proof_sha256"):
+        raise NativeSeq32AttentionMlpSingleProofGateError("proof sha drift")
+    envelope_raw = read_repo_file(SINGLE_ENVELOPE, "native single envelope")
+    if sha256(envelope_raw) != row.get("envelope_sha256"):
+        raise NativeSeq32AttentionMlpSingleProofGateError("envelope sha drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)